src/test/java/net/masterthought/cucumber/ReportParserTest.java

```diff
@@ -88,4 +90,145 @@ public void parseJsonResults_OnNoExistingFile_ThrowsException() {
 
         reportParser.parseJsonFiles(jsonReports);
     }
+
+    @Test
+    public void parsePropertyFiles_EmptyProperties() {
+
+        // given
+        initWithProperties(EMPTY_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).isEmpty();
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_One_File() {
+
+        // given
+        initWithProperties(SAMPLE_ONE_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(5);
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_Two_Files() {
+
+        // given
+        initWithProperties(SAMPLE_ONE_PROPERTIES, SAMPLE_TWO_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(8);
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_Check_Contents_Integrity() {
+
+        // given
+        initWithProperties(SAMPLE_TWO_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        List<Map.Entry<String, String>> classifications = configuration.getClassifications();
```

Comment:
is it // then ?

Reply:
it's a when so I'll update so that they are grouped together